src/Api/Response/ResponseParser.php

```diff
@@ -123,7 +134,7 @@ protected function parseLandingPage(\SimpleXMLElement $data)
     {
         $landingPage = [];
 
-        if (!empty($data->landingPage) ) {
+        if (isset($data->landingPage) && !empty($data->landingPage->attributes()) ) {
```

Comment:
Since there is no apparent reason to return an array (anymore): Return the link as string or `null` if no link was provided.

Reply:
This method still returns an array when it actually should return the string representation of `$data->landingPage->attributes()->link` or `null`. Don't forget to update the tests and PHPDocs.